pulseapi/profiles/serializers.py

```diff
@@ -96,6 +96,11 @@ class UserProfileSerializer(serializers.ModelSerializer):
         allow_blank=True,
     )
 
+    is_active = serializers.BooleanField(
+        required=False,
```

Comment:
let's make this read_only as well.

Reply:
you also need to remove this from the `exclude` option in the `Meta` class

Reply:
done